src/index.ts

```diff
@@ -69,7 +70,10 @@ async function handleUpload(
     const containerClient = blobSvcClient.getContainerClient(trimParam(config.containerName));
     const client = containerClient.getBlockBlobClient(getFileName(config.defaultPath, file));
     const options = {
-        blobHTTPHeaders: { blobContentType: file.mime },
+        blobHTTPHeaders: { 
+            blobContentType: file.mime,
+            blobCacheControl: trimParam(config.defaultCacheControl)
```

Comment:
Should this have a default/fallback value?

Reply:
I'm thinking no, since docs state you can leave this empty. @jakeFeldman

Reply:
Sounds good. I'll merge these changes now and try to publish asap. Thanks